cmd/world/main.go

```diff
@@ -19,6 +19,7 @@ var (
 	AppVersion    string
 	PosthogAPIKey string
 	SentryDsn     string
+	Env           string
```

Comment:
*optional*: add `-X main.Env=<DEV|PROD>` to the go build example comment above the var declaration, since the other values are also there

Reply:
done